src/libpriv/rpmostree-core.cxx

```diff
@@ -1092,7 +1093,7 @@ rpmostree_context_download_metadata (RpmOstreeContext *self,
    * https://github.com/projectatomic/rpm-ostree/issues/1127
    */
   if (self->rojig_pure)
-    flags |= DNF_CONTEXT_SETUP_SACK_FLAG_SKIP_FILELISTS;
+    flags = static_cast<DnfContextSetupSackFlags>(static_cast<int>(flags) | DNF_CONTEXT_SETUP_SACK_FLAG_SKIP_FILELISTS);
```

Comment:
I hope there's a better answer than this or https://stackoverflow.com/a/10339152/12240197